kv/store.go

```diff
@@ -14,7 +14,14 @@ import (
 	"github.com/serverless/event-gateway-connector/connection"
 )
 
-const jobsDir = "jobs/"
+const (
+	prefix            = "serverless-event-gateway-connector/"
+	connectionsPrefix = "connections/"
+	locksPrefix       = "locks/jobs/"
+	checkpointPrefix  = "workers/"
```

Comment:
what about `checkpoints`? :)

Reply:
it's called `checkpointPrefix`, but you'd rather call it `checkpoints`?